pkgs/applications/version-management/gitlab/gitaly/default.nix

```diff
@@ -24,7 +24,7 @@ let
       owner = "gitlab-org";
       repo = "gitaly";
       rev = "v${version}";
-      sha256 = "sha256-Tc7JsGuTF+JK5SLmaQwVbmdSy/Wvm3VNoZqHTyZFd6k=";
+      sha256 = "sha256-Q2wXoc4tLJvc0LwSct3cCswOqhUJSVVhoG/uFXeVOB4=";
```

Comment:
```suggestion
      hash = "sha256-Q2wXoc4tLJvc0LwSct3cCswOqhUJSVVhoG/uFXeVOB4=";
```

Reply:
These are automatically generated through the update script.

Reply:
nix-update only exchanges the value, but works with both `sha256` and `hash` and `hash` is the new one you are supposed to use now.